clarity-ext-scripts/clarity_ext_scripts/covid/label_printer.py

```diff
@@ -48,7 +48,9 @@ def append_contents(self, label_print_info):
 
     @property
     def contents(self):
-        return '\n'.join(self.printer.contents)
+        content = '\n'.join(self.printer.contents)
+        content = ''.join(["${", content, "}$"])
+        return content
```

Comment:
Logic change. Look at this.

Reply:
Looks like a very clean fix. I'll try it on stage later today, before I leave the office.

Reply:
We have always tested the barcodes with a newline after "${" and before "}$". I'm convinced that this will work as well. Perhaps though it's best to not take any chances?

Reply:
Ok. I'll add newlines before and after.

Reply:
I've never put any newlines in when I've tested this manually on the printer, so I'm 100% sure it works, but I like your idea, because it makes it much easier to copy paste the ZPL code from the file without having to bother with manually removing the pre- and postfixes.

Reply:
It works both, with new line and without. I have checked. But better to have it.